react-user-tour/react-user-tour.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for react-user-tour
+// Project: https://github.com/socialtables/react-user-tour
+// Definitions by: Carlo Cancellieri <https://github.com/ccancellieri>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "react-user-tour" {
```

Comment:
You need to add a reference to the `react.d.ts` to import.
Also, use 4 spaces instead of 2 for consistence